src/utils/hearing.js

```diff
@@ -0,0 +1,32 @@
+import _ from 'lodash';
+
+
+export function getMainSection(hearing) {
+  return _.find(hearing.sections, (section) => section.type === "main");
+}
+
+/*
+* Return URL to hearing view. Accepts optional fullscreen parameter
+* to force fullscreen query parameter.
+ */
+export function getHearingURL(hearing, {fullscreen} = {}) {
+  const url = `/hearing/${hearing.id}`;
```

Comment:
This should probably to be fixed to `/hearing/${hearing.id}`instead? Currently, redirect from hearing/slug to hearing/slug?fullscreen=true does not work, because we are making the assumption that the original url was of the form hearing/id.

Also, this fix would have the admittedly handy side effect of fixing issue #247 =)